packages/components/date-input/README.md

```diff
@@ -88,10 +90,97 @@ Find out more about the date input component and when to use it in the [NHS digi
+
+[Preview the date input with autocomplete attribute component](https://nhsuk.github.io/nhsuk-frontend/components/date-input/autocomplete.html)
+
+#### HTML markup
+
+```html
+<div class="nhsuk-form-group">
+  <fieldset class="nhsuk-fieldset" aria-describedby="dob-with-autocomplete-attribute-hint" role="group">
+    <legend class="nhsuk-fieldset__legend">
+      What is your date of birth?
+    </legend>
+    <span class="nhsuk-hint" id="dob-with-autocomplete-attribute-hint">
+    For example, 31 3 1980
+    </span>
+    <div class="nhsuk-date-input" id="dob-with-autocomplete-attribute">
+      <div class="nhsuk-date-input__item">
+        <div class="nhsuk-form-group">
+          <label class="nhsuk-label nhsuk-date-input__label" for="dob-with-autocomplete-attribute-day">
+          Day
+          </label>
+          <input class="nhsuk-input nhsuk-date-input__input nhsuk-input--width-2" id="dob-with-autocomplete-attribute-day" name="dob-with-autocomplete-day" type="number" autocomplete="bday-day" pattern="[0-9]*">
+        </div>
+      </div>
+      <div class="nhsuk-date-input__item">
+        <div class="nhsuk-form-group">
+          <label class="nhsuk-label nhsuk-date-input__label" for="dob-with-autocomplete-attribute-month">
+          Month
+          </label>
+          <input class="nhsuk-input nhsuk-date-input__input nhsuk-input--width-2" id="dob-with-autocomplete-attribute-month" name="dob-with-autocomplete-month" type="number" autocomplete="bday-month" pattern="[0-9]*">
+        </div>
+      </div>
+      <div class="nhsuk-date-input__item">
+        <div class="nhsuk-form-group">
+          <label class="nhsuk-label nhsuk-date-input__label" for="dob-with-autocomplete-attribute-year">
+          Year
+          </label>
+          <input class="nhsuk-input nhsuk-date-input__input nhsuk-input--width-4" id="dob-with-autocomplete-attribute-year" name="dob-with-autocomplete-year" type="number" autocomplete="bday-year" pattern="[0-9]*">
+        </div>
+      </div>
+    </div>
+  </fieldset>
+</div>
+```
+
+#### Nunjucks macro
+
+```
+{% from 'components/date-input/macro.njk' import dateInput %}
+
+{{ dateInput({
+  "id": "dob-with-autocomplete-attribute",
+  "namePrefix": "dob-with-autocomplete",
+  "fieldset": {
+    "legend": {
+      "text": "What is your date of birth?"
+    }
+  },
+  "hint": {
+    "text": "For example, 31 3 1980"
+  },
+  "items": [
+    {
+      "name": "day",
+      "classes": "nhsuk-input--width-2",
+      "autocomplete": "bday-day"
+    },
+    {
+      "name": "month",
+      "classes": "nhsuk-input--width-2",
+      "autocomplete": "bday-month"
+    },
+    {
+      "name": "year",
+      "classes": "nhsuk-input--width-4",
+      "autocomplete": "bday-year"
+    }
+  ]
+}) }}
+```
+
+---
+
 ### Date input with errors
 
 [Preview the date input with errors component](https://nhsuk.github.io/nhsuk-frontend/components/date-input/multiple-errors.html)
 
+#### Guidance
```

Comment:
@AdamChrimes - should this be further up, around line 97?

Reply:
@mcheung-nhs oops, yep. Moved it up.